gen/calc.py

```diff
@@ -113,6 +113,11 @@ def validate_url(url: str):
         ) from ex
 
 
+def validate_absolute_path(path):
```

Comment:
Mhm. Can we rename this to `validate_startswith_slash(text)`?

Reply:
Ha well, I see that the error message should be adjusted to the case where this is a filesystem path. But `/` as well as `/etc` both pass this validation, and I wonder if we're actually checking for whatever matters most here.

Reply:
Should we maybe additionally confirm that
* this does not have a trailing slash
* this results in a non-emtpy dirname and non-empty basename when using `os.path.split()` on the text?

Reply:
Here's a successful integration test run that specifies the work dirs with trailing slashes: https://teamcity.mesosphere.io/viewLog.html?buildId=808547&buildTypeId=DcOs_Open_Test_IntegrationTest_AwsOnpremWStaticBackend&tab=buildResultsDiv

Agreed that we should catch as many problems as possible prior to deployment, since that's when they're easier to debug. But I don't think it's worth it to validate the location itself, since the idea behind this feature is to allow users to adapt their installation to meet requirements imposed by their environment that we can't predict. I think it'd make more sense to explain in documentation what these params do, that they're intended for advanced users, and that they should be left at their defaults unless there's a specific reason to change them.